packages/flutter/lib/src/material/input.dart

```diff
@@ -13,6 +13,126 @@ import 'theme.dart';
 
 export 'package:sky_services/editing/editing.mojom.dart' show KeyboardType;
 
+/// A container for grouping together multiple Input widgets.
+class Form extends StatefulWidget {
```

Comment:
Can we put Form (and its associate classes) in their own file?  We'll probably want to extend this over time to handle checkboxes and such.  When we do that, we'll want it to be loosely coupled with the Input widget.

Actually, I wonder if we could put the Form widget in the widgets layer rather than the material layer.  It doesn't seem specific to material design.  It's fine to use the Input widget to wire up to it, but if/when we add another design language, we'll probably want to share/re-use Form.